blocks/components/editable/tinymce.js

```diff
@@ -19,12 +22,42 @@ export default class TinyMCE extends wp.element.Component {
 		}
 	}
 
-	shouldComponentUpdate() {
+	componentDidMount() {
+		this.initialize();
+	}
+
+	shouldComponentUpdate( nextProps ) {
 		// We must prevent rerenders because TinyMCE will modify the DOM, thus
 		// breaking React's ability to reconcile changes.
 		//
 		// See: https://github.com/facebook/react/issues/6802
-		return false;
+		return nextProps.tagName !== this.props.tagName;
```

Comment:
I'm surprised to find that this doesn't really break. I think it'd be worth updating the preceding comment to describe when and why we allow a rerender, and ideally how it's exempt from the concern noted in the referenced issue.